frontend/src/App.tsx

```diff
@@ -56,6 +59,21 @@ const App = (): React.ReactElement => {
               <Switch>
                 <Route exact path={Routes.LOGIN_PAGE} component={Login} />
                 <Route exact path={Routes.SIGNUP_PAGE} component={Signup} />
+                <PrivateRoute
+                  exact
+                  path={Routes.CAMPS_PAGE}
+                  component={CampsPage}
+                />
+                <PrivateRoute
+                  exact
+                  path={Routes.GLOBAL_FORMS_PAGE}
+                  component={GlobalFormsPage}
+                />
+                <PrivateRoute
+                  exact
+                  path={Routes.ACCESS_CONTROL_PAGE}
+                  component={AccessControlPage}
+                />
                 <PrivateRoute
                   exact
                   path={Routes.HOME_PAGE}
```

Comment:
Right now the `HOME_PAGE` route at `/` is set to the Default page component, change it to the CampsPage here